docs/aurora/data-science/python.md

```diff
@@ -1,5 +1,10 @@
 # Python on Aurora
 
+> [!IMPORTANT]
```

Comment:
I didnt check the rendering--- does this conform to https://squidfunk.github.io/mkdocs-material/reference/admonitions/ ?